network-api/networkapi/wagtailpages/templates/wagtailpages/bannered_campaign_page.html

```diff
@@ -8,3 +8,19 @@
 {% endblock %}
 
 {% block campaign_page_header %}{% endblock %}
+{% block campaign_page_menu %}{% endblock %}
+
+{% block bannered_campaign_page_menu %}
+{% comment %}
+  Wrapper HTML elements copied from primary_page_menu.html as we want menu on BanneredCampaignPage to look exactly the same as PrimaryPage's.
```

Comment:
If you put this _above_ the block, then it's "not templating content" and you don't need the comment tag wrapping.